demo/config.yaml

```diff
@@ -1,10 +1,10 @@
 lcr-modules:
 
     _shared:
-        lcr-modules: "../"
-        lcr-scripts: "../../lcr-scripts"
+        lcr-modules: "/projects/rmorin/projects/tumour_contam/lcr-modules/"
+        lcr-scripts: "/projects/rmorin/projects/tumour_contam/lcr-scripts/"
         root_output_dir: "results/"
-        scratch_directory: "scratch/"
+        scratch_directory: "/projects/rmorin_scratch/jawong_tmp/"
```

Comment:
It would be easier to just symlink "/projects/rmorin_scratch/jawong_tmp/" to ./scratch so you don't have to change the config

Reply:
Reverted back